README.md

```diff
@@ -58,6 +58,7 @@ $ sudo apt-get install mycli # Only on debian or ubuntu
       -S, --socket TEXT             The socket file to use for connection.
       -p, --password TEXT           Password to connect to the database.
       --pass TEXT                   Password to connect to the database.
+      --pass PATH                   Password contained in a file or FIFO to connect to the database.
```

Comment:
it still says `--pass`

Reply:
my bad, I didn't commit the README.md file correctly